plugins/katello/nightly/user_guide/content_view_import_export/index.md

```diff
@@ -100,38 +100,42 @@ The `hammer content-view version export` command gathers information about a con
 
-:::warning
-You will need to make sure Katello and Pulp can both read the tar file. If it cannot, you may get an error. The error will be logged in `/var/log/foreman/production.log` for Katello, or in `/var/log/messages` for Pulp.
-:::
+* The `import` command will synchronize the packages from the export tar file into Library. It will then call the `/publish` API and create a new content view version using those packages.
 
-:::info
-_NOTE_: The import process will import all errata from an export. It will then purge any errata that are not associated with packages. This is the same process that is used today when copying RPMs between repositories.
-:::
+<div class="alert alert-danger" markdown="1">
+You will need to make sure Katello and Pulp can both read the tar file. If it cannot, you may get an error. The error will be logged in **/var/log/foreman/production.log** for Katello, or in **/var/log/messages** for Pulp. Ownership of the directory and files should be **_Apache_** with **_system_u:object_r:httpd_sys_rw_content_t:s0_** as the SELinux context.
+</div>
 
+<div class="alert alert-info" markdown="1">
+**Note**
 
+The import process will import all errata from an export. It will then purge any errata that are not associated with packages. This is the same process that is used today when copying RPMs between repositories.
+</div>
 
 ## Import/Export Best Practices
 
-The intent of import/export is to capture a content view version on one Katello, and then re-create it on another Katello. The feature does *not* replicate a standard operating environment (SOE) from one Katello to another. A standard operating environment includes a manifest file, content view definitions, products, repositories, activation keys, host groups, and other information.
+* The intent of import/export is to capture a content view version on one Katello, and then re-create it on another Katello. The feature does *not* replicate a standard operating environment (SOE) from one Katello to another. A standard operating environment includes a manifest file, content view definitions, products, repositories, activation keys, host groups, and other information.
 
-Please use [foreman-ansible-modules](https://github.com/theforeman/foreman-ansible-modules) or Hammer scripts to define your SOE in a reproducible way. Once you have a reproducible SOE, you can then use import/export to keep your Katello updated.
+* Please use [foreman-ansible-modules](https://github.com/theforeman/foreman-ansible-modules) or Hammer scripts to define your SOE in a reproducible way. Once you have a reproducible SOE, you can then use import/export to keep your Katello updated.
 
+* To ensure proper SELinux contexts on the exporting tar and files, use the `/var/lib/pulp/katello-export` export directory. This directory already has the correct permissions and correct SELinux labels, and was created specifically as a landing place for files not created by Pulp that Pulp needs to read or write.
```

Comment:
`/var/lib/pulp/katello-export` is needed more when importing than when exporting. The user is able to export anywhere, but the tar file(s) need to be in `/var/lib/pulp/katello-export` when importing.

I would also mention again that the alternative is to set file permissions and selinux labels if the user chooses to use a different directory.

Reply:
@beav updated